docs/core/diagnostics/cli-tools/dotnet-counters.md

```diff
@@ -0,0 +1,134 @@
+---
+title: dotnet-counters - .NET Core
+description: Installing and using the dotnet-counter command-line tool.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 08/21/2019
+---
+# dotnet-counters
+
+## Intro
+
+`dotnet-counters` is a performance monitoring tool for ad-hoc health monitoring or first-level performance investigation. It can observe performance counter values that are published via the `EventCounter` [API](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventcounter). For example, you can quickly monitor things like the CPU usage or the rate of exceptions being thrown in your .NET Core application to see if there's anything suspicious before diving into more serious performance investigation using `PerfView` or `dotnet-trace`.
+
+## Install dotnet-counters
+
+To install the latest release version of the `dotnet-counters` [NuGet package](https://www.nuget.org/packages/dotnet-counters):
+
+```
+dotnet tool install --global dotnet-counters
```

Comment:
Are we going to hold publishing this until an RTM version of the tool is released? If not I think we'd need to provide install instructions that work for the Preview bits and then we update it later once final bits are available.

Reply:
I was leaving the installing, upgrading and installing of preview versions to the installer page.

I am hoping we will RTM soon so this isn't really an issue.

Reply:
I wouldn't be surprised if the RTM of the tools is at least a month out, potentially more. I think we've mostly been de-prioritizing issues in the tools so that we could handle runtime stuff, and with bug reports/investigation requests continuing to come in for the runtime its not clear how quickly we'll be able to transition to spending the time we need on the tools.